app/services/pagelets/pagelet.rb

```diff
@@ -0,0 +1,19 @@
+module Pagelets
+  class Pagelet
+    attr_reader :name, :partial, :priority
+
+    def initialize(name, partial, priority)
+      @name = name
+      @partial = partial
+      @priority = priority
+    end
+
+    def <=>(other)
+      other.priority <=> priority
```

Comment:
I guess you just need to swap these guys in order to get the ascending (more logical) order.

* 0 - first element
* 100 - second
* etc